internal/integrations/v4/runner/runner.go

```diff
@@ -218,6 +218,12 @@ func (r *runner) logErrors(errs <-chan error) {
 
 func (r *runner) handleLines(stdout <-chan []byte, extraLabels data.Map, entityRewrite []data.EntityRewrite) {
 	for line := range stdout {
+		// log process stdout when debug mode is active
+		if !r.Integration.IsIntegration {
+			r.log.Debug(string(line))
```

Comment:
Please don't add entries to the log without any message or context.
We could enhance this one like:
`r.log.WithFields("line", string(line)).WithFields("cmd", r.Integration.Name).Debug("non integration cmd")`

Reply:
I saw that the integration name is already added. I will add the "line" field 
This is how now is being print
`time="2020-10-07T10:52:29Z" level=debug msg="stdout line" component=integrations.runner.Runner integration_name=basic_cmd`

Reply:
I was looking at this and the output would differ with the one printed for the stderr in line 188, and brings some confusion when you see both lines:
```
time="2020-10-07T13:12:38Z" level=debug msg="cmd stdout" component=integrations.runner.Runner integration_name=basic_cmd line="stdout line payload"
time="2020-10-07T13:12:38Z" level=debug msg="stderr line payload" component=integrations.runner.Runner integration_name=basic_cmd
```
I would keep same format for both and I don't know if modifying the current format for the stderr could cause some braking change. wdyt? @varas